### PR TITLE
Proxy focus and blur methods

### PR DESCRIPTION
**Pull request by vslinko**

### CODE REVIEW

**Comment by iamdustan** [comment]
Could you describe what this accomplishes and the motivation?

File: src/index.js [hunk 1]
```diff
@@ -160,6 +160,14 @@ var MaskedInput = React.createClass({
     var value = this.mask.getValue()
     return value === this.mask.emptyValue ? '' : value
   },
+  
+  focus() {
+    this.input.focus();
```
Comment (iamdustan) [inline src/index.js:165]: should any arguments like the event object be passed through?

Comment (vslinko) [inline src/index.js:165]: As I know `HTMLInputElement.focus` has no arguments.

**Comment by vslinko** [comment]
I have some cases when I should focus or blur manually, from a parent component.

**Comment by iamdustan** [comment]
ah. so this is to get a ref to the maskedinput so you can do something like:

```
componentWillMount() {
  this.mask && this.mask.focus()
}
render () {
  return <MaskedInput ref={comp = this.mask = comp{ />
}
```

and the MaskedInput will behave as though it was a standard form input?

**Comment by vslinko** [comment]
Yes, you're right.

**Comment by iamdustan** [comment]
:thumbsup: thanks!

**Pull request merged**